subt_ign/src/ConnectionHelper.cc

```diff
@@ -23,6 +23,29 @@
 std::map<std::string, std::vector<ignition::math::Vector3d>>
   subt::ConnectionHelper::connectionPoints =
   {
+    {"Constrained Tunnel Tile Short", {{0, -10, 0}, {0, 10, 0}}},
+    {"Constrained Tunnel Tile Tall", {{0, -10, 0}, {0, 10, 0}}},
+    {"Tunnel Bend Right", {{0, 0, 0}, {17.5, 25, 0}}},
+    {"Tunnel Corner Left", {{0, 0, 0}, {-10, 10, 0}}},
+    {"Tunnel Corner Right", {{0, 0, 0}, {10, 10, 0}}},
+    {"Tunnel Elevation", {{0, 0, 0}, {0, 10, 5}}},
+    {"Tunnel Intersection", {{0, 0, 0}, {7.5, 7.5, 0}, {-7.5, 7.5, 0}, {0, 15, 0}}},
+    {"Tunnel Intersection T", {{0, 0, 0}, {7.5, 7.5, 0}, {-7.5, 7.5, 0}}},
+    {"Tunnel Straight", {{0, 0, 0}, {0, 5, 0}}},
+    {"subt_tunnel_staging_area", {{10, 0, 0}}},
+    {"Tunnel Tile 1", {{0.0, 10.0, 0.0}, {10.0, 0.0, 0.0}, {0.0, -10.0, 0.0}, {010.0, 0.0, 0.0}}},
```

Comment:
There is a duplicated connection point in Tunnel Tile 1, 3, and 4.  Is this intentional?

Reply:
Fixed in 57594ea441af8f3d6a998507115b50d997f6b73c